test/tests/feedTest.js

```diff
@@ -0,0 +1,156 @@
+		});
+		it("should save a feed item with all fields set", function *() {
+			let feed = new Zotero.Feed(),
+				refreshInterval = 30,
+				cleanupAfter = 2;
+			
+			feed.name = name;
+			feed.url = url;
+			feed.refreshInterval = refreshInterval;
+			feed.cleanupAfter = cleanupAfter;
+			
+			let id = yield feed.saveTx({skipEditCheck: true});
+			feed = yield Zotero.Feeds.getAsync(id);
+			feeds.push(feed);
+			
+			assert.equal(feed.name, name, "name is correct");
+			assert.equal(feed.url, url, "url is correct");
+			assert.equal(feed.refreshInterval, refreshInterval, "refreshInterval is correct");
+			assert.equal(feed.cleanupAfter, cleanupAfter, "cleanupAfter is correct");
+			
+			assert.equal(feed.libraryID, Zotero.Libraries.feedLibraryID, "library ID is correct");
+			assert.equal(feed.id, id, "ID is available");
+			assert.isUndefined(feed.lastCheck, "lastCheck is null");
+			assert.isUndefined(feed.lastUpdate, "lastUpdate is null");
+			assert.isUndefined(feed.lastCheckError, "lastCheckError is null");
+		});
+		it("should not allow setting a feed as child of another feed", function *() {
+			let feed = new Zotero.Feed();
+			
+			assert.throws(() => feed.parentID = 1, 'Cannot set parent key for feed', 'cannot set parent by ID');
+			assert.throws(() => feed.parentKey = 1, 'Cannot set parent key for feed', 'cannot set parent by key');
+		});
+		it("should not allow saving a feed with the same url", function *() {
+			let feed1 = new Zotero.Feed();
+			feed1.name = 'Foo';
+			feed1.url = url;
+			
+			let id = yield feed1.saveTx({skipEditCheck: true});
+			feeds.push(yield Zotero.Feeds.getAsync(id));
+			
+			let feed2 = new Zotero.Feed();
+			feed2.name = 'Bar';
+			feed2.url = url;
+			
+			yield assert.isRejected(feed2.saveTx({skipEditCheck: true}));
+		});
+		it("should allow saving a feed with the same name", function *() {
+			let feed1 = new Zotero.Feed();
+			feed1.name = name;
+			feed1.url = 'http://www.example.com';
+			
+			feed1 = yield Zotero.Feeds.getAsync(yield feed1.saveTx({skipEditCheck: true}));
+			feeds.push(feed1);
+			
+			let feed2 = new Zotero.Feed();
+			feed2.name = name;
+			feed2.url = 'http://www.zotero.org';
+			
+			feed2 = Zotero.Feeds.getAsync(yield feed2.saveTx({skipEditCheck: true}));
+			yield assert.isFulfilled(feed2, 'allow saving feed with an existing name');
+			feed2 = yield feed2;
+			feeds.push(feed2);
+			
+			assert.equal(feed1.name, feed2.name, 'feed names are the same');
+		});
+	});
+	describe.skip("#getExpiredFeedItemIDs()", function() {
+	});
+	describe.skip("#updateFeed()", function() {
+	});
+	describe("#erase()", function() {
+		it("should erase a saved feed", function *() {
+			let feed = new Zotero.Feed();
+			feed.name = 'Foo';
+			feed.url = 'http://www.exampled.com';
+			let id = yield feed.saveTx({skipEditCheck: true});
+			feed = yield Zotero.Feeds.getAsync(id);
+			yield feed.eraseTx();
+			
+			yield assert.isRejected(feed.eraseTx(), "does not allow erasing feed twice");
```

Comment:
This currently does not thorw/rejects and I guess it's not a big deal, but IMO it should be an error to try and delete an object twice.